app/Console/Command/ConvertDescriptionToEducationalDescriptionForCultuurkuur.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Organizer\Commands\ConvertDescriptionToEducationalDescription;
+use CultuurNet\UDB3\Search\OrganizersSapi3SearchService;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ConvertDescriptionToEducationalDescriptionForCultuurkuur extends AbstractCommand
+{
+    private const QUERY_LABEL = 'labels:cultuurkuur';
```

Comment:
The label CultuurKuur uses is `cultuurkuur_organizer`, so this should be changed.